integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/bundle/BundledReadAlarmRegister.feature

```diff
@@ -8,13 +8,24 @@ Feature: SmartMetering Bundle - ReadAlarmRegister
   I want to be able to read the alarm register from a meter via a bundle request
 
   Background: 
+
+  Scenario Outline: Retrieve events of a device in a bundle request
     Given a dlms device
       | DeviceIdentification | TEST1024000000001 |
       | DeviceType           | SMART_METER_E     |
-
-  Scenario: Retrieve events of a device in a bundle request
-    Given a bundle request
+      | SelectiveAccessSupported | true              |
+      | Protocol                 | <protocol>        |
+      | ProtocolVersion          | <version>         |
+      | Port                     |              1028 |
```

Comment:
Doordat je poor 1028 gebruikt, test hij alleen maar op een 5.1 simulator, deze poort is afahankelijk van het protocol. Als je hem niet specificeert dan pakt de test automatisch de juiste. 
De port moet hier iig niet fixed op 1028 staan

Reply:
ok, port is verwijderd.